Mesh_3/doc/Mesh_3/CGAL/Mesh_3/parameters.h

```diff
@@ -451,6 +451,72 @@ unspecified_type odt(const Named_function_parameters& np = parameters::default_v
 template <class NamedParameters = parameters::Default_named_parameters>
 unspecified_type perturb(const Named_function_parameters& np = parameters::default_values());
 
+/*!
+ * \ingroup PkgMesh3Parameters
+ *
+ * The function `parameters::initial_points_generator()` enables the user to specify a functor that follows
+ * the `InitialPointsGenerator` concept to the mesh generation function `make_mesh_3()`. This functor is called
+ * for the initialization of the meshing process. If this parameter is specified without arguments, the default behavior
+ * is executed, which calls the domain's `construct_initial_points_object()` for the initialization of the meshing process.
+ *
+ * If the parameter `parameters::initial_points()` is used, the points it provides are inserted before any other initialization.
+ *
+ * If the generator does not generate enough points, the domain's `construct_initial_points_object()` will be called.
```

Comment:
"does not generate enough points", I would add "enough point to ..." ("create a non empty complex", or something like this)